src/Microsoft.DocAsCode.HtmlToPdf/ConvertWrapper.cs

```diff
@@ -132,6 +132,19 @@ private void ConvertCore(string basePath)
                             AbsolutePathInTocPageFileTransformer(tocPageFilePath);
                             htmlModels.Insert(0, new HtmlModel { Title = _pdfOptions.TocTitle, HtmlFilePath = tocPageFilePath });
                         }
+
```

Comment:
DocFX generates one PDF for each TOC. So for a single TOC here, we need not find all the cover pages in manifest. We can make this conversion: only the `cover.xxx` that sibling to TOC in the input folder is treated as current PDF's cover page. Then the logic here is like `var coverPage = FindSiblingPage(tocManifestItem)`